lib/unleash/streaming_client_executor.rb

```diff
@@ -10,6 +12,20 @@ def initialize(name, engine)
       self.event_source = nil
       self.event_processor = Unleash::StreamingEventProcessor.new(engine)
       self.running = false
+
+      begin
+        # if bootstrap configuration is available, initialize. Otherwise read backup file
+        if Unleash.configuration.use_bootstrap?
+          bootstrap(engine)
+        else
+          read_backup_file!(engine)
+        end
+      rescue StandardError => e
+        # fail back to reading the backup file
```

Comment:
Also a hangover from the old code haha, but may as well fix it while we're here
```suggestion
        # fall back to reading the backup file
```